server/src/main/java/io/crate/types/DataTypes.java

```diff
@@ -346,6 +350,7 @@ private static boolean safeConversionPossible(DataType type1, DataType type2) {
         entry("string", STRING),
         entry("varchar", STRING),
         entry("character varying", STRING),
+        entry("time with time zone", TIMETZ),
```

Comment:
Shouldn't the type name be `time with time zone`, see https://www.postgresql.org/docs/9.1/datatype-datetime.html and `timetz` would be an alias? 

> ``timetz`` is a valid alias for `time with time zone`.

You have documented it as it supposedly has to be, but it implementation it is another way around.

Reply:
ok, yes, I suppose I need to declare it as `time with time zone`, but make sure that it ends up as `timetz` in [here](https://crate.io/docs/crate/reference/en/latest/interfaces/postgres.html?highlight=postgres#pg-type).